cmake/externals/sentencepieceproject.cmake

```diff
@@ -1,27 +1,33 @@
+# spm is abbreviation of sentencepiece to meet the path length limits on Windows
+
+set(spm_patches)
+
+if(_ONNXRUNTIME_EMBEDDED)
+  # use protobuf provided by ORT
+  list(APPEND spm_patches "${PROJECT_SOURCE_DIR}/cmake/externals/sentencepieceproject.protobuf.patch")
```

Comment:
the original patch does assume the presence of ORT variables.
updating the condition to apply it from `onnxruntime_BUILD_WEBASSEMBLY` to  `_ONNXRUNTIME_EMBEDDED`